index.html

```diff
@@ -5131,92 +5108,36 @@ <h3>The <dfn>JsonLdProcessor</dfn> Interface</h3>
           <li>Create a new <a>Promise</a> <var>promise</var> and return it.
             The following steps are then executed asynchronously.</li>
           <li>If the passed <a data-lt="jsonldprocessor-expand-input">input</a>
-            is a <a>string</a> representing the <a>IRI</a> of a remote document, dereference it as <var>original input</var>.
-            <ol>
-              <li class="changed">If the retrieved document's <a>Content-Type</a> is <code>text/html</code>,
-                and the processor is a <a>pure JSON Processor</a>
-                reject the <var>promise</var> passing a <a data-link-for="JsonLdErrorCode">loading document failed</a> error.</li>
-              <li>Otherwise, if the retrieved document's <a>Content-Type</a> is neither
-                <code>application/json</code>,
-                <code>application/ld+json</code>,
-                nor any other media type using a
-                <code>+json</code> suffix as defined in [[RFC6839]],
-                reject the <var>promise</var> passing a <a data-link-for="JsonLdErrorCode">loading document failed</a> error.</li>
-            </ol>
+            is a <a>string</a> representing the <a>IRI</a> of a remote document, await and dereference it as <var>remote document</var>
+            using <a>LoadDocumentCallback</a>, passing <a data-lt="jsonldprocessor-expand-input">input</a>
+            for <a data-link-for="LoadDocumentCallback">url</a>,
+            the <a data-link-for="JsonldOptions">extractAllScripts</a> option for <a data-link-for="LoadDocumentOptions">extractAllScripts</a>.
           </li>
+          <li class="changed">If necessary, transform <a data-link-for="RemoteDocument">document</a>
```

Comment:
"If necessary" is not very clear to me. I'm guessing it means "if 'document' is a neither a dictionary not an array". This should be made explicit, all the more than a plain string is listed as a valid "internal representation" (which makes sense as the JSON syntax is able to represent a single string).

Reply:
I'll change, but even though a string is a valid JSON document, it's not a valid JSON-LD document, so the intention should be clear.